LDK/src/org/labkey/ldk/LDKController.java

```diff
@@ -12,6 +12,8 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
+ * Update 2020-05-21 added mapping to IM field for use in pulling employee ID
+ * Build by Josh
```

Comment:
This isn't relevant after some other refactors - let's trim this comment.

Reply:
you dont want credit?

Reply:
@labkey-jeckels you did want this removed, right? I just committed it